etna/analysis/forecast/plots.py

```diff
@@ -35,6 +37,26 @@
     from etna.transforms import Transform
 
 
+def _get_borders_comparator(segment_borders: pd.DataFrame) -> Callable[[str, str], int]:
+    """Create comparator function to sort border names."""
+
+    def cmp(name_a: str, name_b: str) -> int:
+        """Compare two series based on their values."""
+        border_a = segment_borders[name_a].values
+        border_b = segment_borders[name_b].values
+
+        if np.all(border_a == border_b):
+            return 0
```

Comment:
As I understand, in this case the borders are intersecting, but we don't raise any error. Are we ok with this?

Reply:
In this case, we consider them identical, no visual discrepancies will occur. This might happen especially with simple interval methods and constant forecasts.